### PR TITLE
data: vishalrathore8oct

### PR DESCRIPTION
**Pull request by vishalrathore8oct**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4349"></a>

### CODE REVIEW

File: data/vishalrathore8oct.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://vishalrathore.hashnode.dev/",
+      "icon": "SiHashnode"
+    },
+    {
+      "name": "Follow me on Showcase",
+      "url": "https://www.showwcase.com/vishalrathore8oct",
+      "icon": "SiShowtime"
+    },
```
Comment (swagataroy30) [inline data/vishalrathore8oct.json:31]:
```suggestion
    }
```

**Review by swagataroy30 — COMMENTED** [review]
Can you please remove the comma from line 31. Thank You!

**Review by Krishnapro — CHANGES_REQUESTED** [review]
please remove `,` after `}` from line 31.

File: data/vishalrathore8oct.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://github.com/vishalrathore8oct",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://vishalrathore.hashnode.dev/",
+      "icon": "SiHashnode"
+    },
+    {
+      "name": "Follow me on Showcase",
```
Comment (rupali-codes) [inline data/vishalrathore8oct.json:28]: It should be `Showwcase`

Comment (rupali-codes) [inline data/vishalrathore8oct.json:28]:
```suggestion
      "name": "Follow me on Showwcase",
```

**Pull request merged**